test/python/test_tools_import_special_phrases.py

```diff
@@ -0,0 +1,174 @@
+"""
+    Tests for import special phrases functions
+"""
+import pytest
+from nominatim.tools.special_phrases import _create_place_classtype_indexes, _create_place_classtype_table, _get_wiki_content, _grant_access_to_webuser, _process_amenity
+
+def test_get_wiki_content():
+    assert _get_wiki_content('fr')
```

Comment:
The unit tests here should avoid retrieving actual data from the Internet. It tends to fail at inconvenient times. It's fine if you keep _get_wiki_content() untested because of this. Where you need an actual page for the tests, download a current version, put it into `test/testdata` and mock _get_wiki_content() to return the saved file.